qualys-web-application-scanner/info.json

```diff
@@ -342,77 +357,79 @@
           "visible": true,
           "type": "datetime",
           "name": "launchedDate",
-          "description": " The date and time when the scan was launched in UTC date/time format (YYYY-MM-DDTHH:MM:SSZ)"
+          "description": "Specify the date and time to return all the scan which was launched after specified date and time in UTC date/time format (YYYY-MM-DDTHH:MM:SSZ)",
+          "tooltip": "Specify the date and time to return all the scan which was launched after specified date and time in UTC date/time format (YYYY-MM-DDTHH:MM:SSZ)"
         },
         {
           "title": "Type",
           "required": false,
           "editable": true,
           "visible": true,
-          "type": "select",
-          "option": [
+          "type": "multiselect",
```

Comment:
Mention the type change in release notes.

Reply:
done